apify-api/openapi/paths/actor-tasks/actor-tasks@{actorTaskId}@run-sync.yaml

```
@@ -209,6 +209,20 @@ post:
         type: number
         format: double
         example: 1000
+    - name: maxTotalChargeUsd
+      in: query
+      description: |
+        Specifies the maximum cost of the task run. This parameter is
+        useful for pay-per-event tasks, as it allows you to limit the amount
+        charged to your subscription. You can access the
+        maximum cost in your Actor by using the
+        `ACTOR_MAX_TOTAL_CHARGE_USD` environment variable.
```

Should we refer to it as `pay-per-event task`? Correct me if I'm wrong but only Actor can be `pay-per-event` and task inherently have no monetization but they can be comprised of Actors right? 

Maybe something like:

```suggestion
        Specifies the maximum cost of the task run. This parameter is
        useful when using pay-per-event Actors, as it allows you to limit the amount
        charged to your subscription. You can access the
        maximum cost in your Actor by using the
        `ACTOR_MAX_TOTAL_CHARGE_USD` environment variable.
```

---

Let's leave it for now, but I might revisit it later.